gradle.properties

```diff
@@ -1,6 +1,6 @@
 kotlinVersion=1.0.6
 junitPlatformVersion=1.0.0-M3
-kotlin.incremental=true
+kotlin.incremental=false
```

Comment:
Ah, we had such problems like half year ago, currently incremental compilation is very stable and damn it saves lots of time!